WooCommerce/WooCommerceTests/ViewRelated/CardPresentPayments/CardPresentPaymentsOnboardingUseCaseTests.swift

```diff
@@ -287,6 +287,27 @@ class CardPresentPaymentsOnboardingUseCaseTests: XCTestCase {
 
     }
 
+    func test_onboarding_sends_install_plugin_action_for_wcpay_plugin_when_installPlugin_is_invoked() throws {
```

Comment:
Not a blocker, but we could use the `when/then` syntax here

Reply:
Sounds good! Done here https://github.com/woocommerce/woocommerce-ios/pull/10580/commits/99c93b8e4bc55937a538ec7791804696b361248c